.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@0.3.2
+
+jobs:
+  manylinux2014-aarch64:
+    parameters:
+      cibw_build:
+        type: string
+
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    steps:
+      - checkout
+      - run:
+          name: Build manylinux AArch64 wheel
+          command: |
+            docker run --rm \
+                -w /root/nrn \
+                -v $PWD:/root/nrn \
+                -v /opt/nrnwheel/mpt:/nrnwheel/mpt \
+                -e NEURON_NIGHTLY_TAG \
+                -e NRN_NIGHTLY_UPLOAD \
+                -e NRN_RELEASE_UPLOAD \
+                -e NEURON_WHEEL_VERSION \
+                -e NRN_BUILD_FOR_UPLOAD=1 \
+                'odidev/neuron_wheel' \
+                packaging/python/build_wheels.bash linux << parameters.cibw_build >> coreneuron
+      - store_artifacts:
+          path: ./wheelhouse
+          destination: artifacts
+
```

Comment:
Once the wheel is built, it would be great if we could run minimal tests to make sure wheel is functional. For example, we run azure tests [here](https://github.com/neuronsimulator/nrn/blob/d0a5d6f8cd2e545e82303f0b51683a9c165d392d/ci/azure-wheel-test-upload.yml#L22).